Apply correct grid markup to cookie page

owenatgov opened the pull request:
## What
Reworks how the grid is used on the cookie page to be semantically correct.

## Why
Fixes https://github.com/alphagov/govuk_publishing_components/issues/2202

## Visual changes (no JS)
### Before
![www gov uk_help_cookies](https://user-images.githubusercontent.com/64783893/145857985-9dd62321-911b-4fdc-847a-2612ba2c320a.png)


### After
![frontend dev gov uk_help_cookies](https://user-images.githubusercontent.com/64783893/145858012-5dbdf8fc-27c5-4749-9ba8-2ef1d32a267b.png)

andysellick commented on app/views/help/cookie_settings.html.erb:
```diff
@@ -1,13 +1,16 @@
       } %>
   </div>
 
+  <div class="govuk-grid-column-two-thirds">
```
Why did you add another grid column here? It's the same as the one above - would all of the content not fit into that one?

owenatgov commented:
@andysellick I've addressed your review and also moved a few classes around as the way we were using `main` here was inconsistent with how we use it elsewhere (no classes, only the content id and `role="main"`) and removed a few redundant classes which I suspect are going to break a few smokey tests but I can fix them upstream. Take a look when you have a sec.

owenatgov commented:
We've already got a failing test! Investigating.

andysellick reviewed (approved):
Nice 🎄
